packages/interaction-kit/src/structures/markdown.ts

```diff
@@ -25,6 +25,10 @@ export function emoji(emoji: { name: string } & Mentionable, animated = false) {
 	return `<:${emoji.name}:${emoji.id}>`;
 }
 
+export function command({ id, name }: { name: string } & Mentionable) {
```

Comment:
There should probably be a way to more gracefully handle subcommands since if you want to mention a subcommand you'd have to pass in the top level command's id and then the full name with the top level name and subcommand/group. Maybe have a couple more optional args for `group` and `subcommand`, and then inside the function those get added to the name with spaces? Something like this potentially?
```ts
export function command({ id, name }: { name: string } & Mentionable): string
export function command({ id, name }: { name: string } & Mentionable, subcommandOrGroup?: string): string
export function command({ id, name }: { name: string } & Mentionable, group?: string, subcommand?: string): string
export function command({ id, name }: { name: string } & Mentionable, subcommandOrGroup?: string, subcommand?: string): string {
    const fullName = [name, subcommandOrGroup, subcommand].filter(piece => piece !== undefined).join(" ");
    return `</${fullName}:${id}>`;
}
```

That way a user can always pass the top level command object and then the subcommand/group as the next two arguments for instance `command(MyTopLevelCommand, "group", "subcommand")`

Reply:
if I'm being honest I forgot you could mention subcommands

Reply:
Took this idea and tweaked it a bit!